README.md

```diff
@@ -14,14 +14,14 @@ for applicable air flight routes from a previously populated database. An additi
 ## Prerequisites
 The following pieces need to be in place in order to run the application.
 
-* Couchbase Server 5.0 or later with the travel-sample bucket setup
+* Couchbase Server 6.5 (Developer Preview) or later with the travel-sample bucket setup
 * Couchbase C Library -
 
 - *Windows Users* :This library is bundled with the Python SDK installer. The installer can be downloaded from [Pypi](https://pypi.python.org/pypi/couchbase) website. Scroll to the section which lists the relevant installers
    ![](https://raw.githubusercontent.com/couchbaselabs/try-cb-python/5.0/win_py_installer.png)
-- *Other Platform Users*: Download [libcouchbase](https://developer.couchbase.com/documentation/server/current/sdk/c/start-using-sdk.html) for platform of choide
+- *Other Platform Users*: Download [libcouchbase](https://developer.couchbase.com/documentation/server/current/sdk/c/start-using-sdk.html) for your chosen platform
```

Comment:
It appears this now depends on PYCBC 3.x, which means that by default it will attempt to pull in and build libcouchbase itself, or pull in a binary wheel of LCB+PYCBC if available (just for Windows as is). It is still possible to self-build LCB by setting the env var PYCBC_BUILD to 'distutils', but this wouldn't be the recommended method for non-advanced users. Bear in mind for the 'bundled' build of LCB included with PYCBC by default you still need LCB's build prerequisites. We should probably update PYCBC 3.x's build notes to cover this if not already added - there is a JIRA for this.